doc/track_2_migration_guidance.md

```diff
@@ -24,7 +24,7 @@ Below is a list of typical issues.
 
 ### Long running operation function name change
 
-Long running operations have changed their function names in Track 2 SDK. A `begin_` prefix is added. E.g. `create_or_update` becomes `begin_create_or_update`. `delete` becomes `begin_delete`. It is a naming convention in Track 2 SDK to indicate that an operation is a long running operation.
+Long running operations have changed their function names in Track 2 SDK. A `begin_` prefix is added. E.g. `create_or_update` becomes `begin_create_or_update`. `delete` becomes `begin_delete`. It is a naming convention in Track 2 SDK to indicate that an operation is a long running operation. Test cases can uncover most instances. But if a command has no test, it will be missed. A rigorous method is going through all methods to see whether they are long running operations.
```

Comment:
I prefer the word "reveal", because "uncover" is usually for something secret:

https://www.collinsdictionary.com/us/dictionary/english/uncover

> If you **uncover** something, especially something that has been kept secret, you discover or find out about it.

```suggestion
Long running operations have changed their function names in Track 2 SDK. A `begin_` prefix is added. E.g. `create_or_update` becomes `begin_create_or_update`. `delete` becomes `begin_delete`. It is a naming convention in Track 2 SDK to indicate that an operation is a long running operation. Test cases can reveal most instances, but if a command has no test, it may be missed. A reliable approach is going through all methods to see whether they are long running operations.
```

Reply:
"e.g." is usually place at the end of a sentence.

https://www.collinsdictionary.com/us/dictionary/english/eg

> We need helpers of all types, e.g., geologists and teachers.

```suggestion
Long running operations have changed their function names in Track 2 SDK. A `begin_` prefix is added. For example, `create_or_update` becomes `begin_create_or_update`. `delete` becomes `begin_delete`. It is a naming convention in Track 2 SDK to indicate that an operation is a long running operation. Test cases can uncover most instances. But if a command has no test, it will be missed. A rigorous method is going through all methods to see whether they are long running operations.
```